stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -53,14 +55,6 @@
  */
 public class StripeApiHandler {
 
-    public static final String LIVE_API_BASE = "https://api.stripe.com";
-    public static final String LIVE_LOGGING_BASE = "https://q.stripe.com";
-    public static final String CHARSET = "UTF-8";
-    public static final String TOKENS = "tokens";
-    public static final String SOURCES = "sources";
```

Comment:
Partly historic reasons (some of them were public when I got here) and otherwise just because I defaulted to keeping the values that were constants available. It was a mistake that I compounded upon by continuing it -- I was thinking in app-dev mode rather than library-dev mode.